src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/generalapplicationdirection/GeneralApplicationDirectionsSubmittedHandler.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler.generalapplicationdirection;
```

Comment:
this package name is different from the one in test
`generalapplicationdirections`